third_party/github.com/juanjoDiaz/removeNPMAbsolutePaths/.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: node_js
```

Comment:
nit: I'd rather just vendor the files that are load bearing or useful as documentation like README and package.json - some of these files are just noise

Reply:
Done